From 0fd36f6aac0c53031eafb32298debf3adc8fb8fe Mon Sep 17 00:00:00 2001
From: Frank Li <Frank.Li@freescale.com>
Date: Mon, 6 Dec 2010 17:14:16 +0800
Subject: [PATCH] ENGR00134193-2 MX28: Fix FEC can't found phy after fix timer issue

mdelay(10) actually delay 50ms before fix timer issue.
After fix timer issue. It should set to 50ms

Signed-off-by: Frank Li <Frank.Li@freescale.com>
---
 arch/arm/mach-mx28/mx28evk_pins.c |    8 +++++++-
 1 files changed, 7 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx28/mx28evk_pins.c b/arch/arm/mach-mx28/mx28evk_pins.c
index 3b6afe4..1990b92 100644
--- a/arch/arm/mach-mx28/mx28evk_pins.c
+++ b/arch/arm/mach-mx28/mx28evk_pins.c
@@ -1120,7 +1120,13 @@ int mx28evk_enet_gpio_init(void)
 	/* reset phy */
 	gpio_request(MXS_PIN_TO_GPIO(PINID_ENET0_RX_CLK), "PHY_RESET");
 	gpio_direction_output(MXS_PIN_TO_GPIO(PINID_ENET0_RX_CLK), 0);
-	mdelay(10);
+
+	/*
+	 * Before timer bug fix(set wrong match value of timer),
+	 * mdelay(10) delay 50ms actually.
+	 * So change delay to 50ms after timer issue fix.
+	 */
+	mdelay(50);
 	gpio_direction_output(MXS_PIN_TO_GPIO(PINID_ENET0_RX_CLK), 1);
 
 	return 0;
-- 
1.5.4.4

